mod_deploy/controllers.py

```diff
@@ -137,10 +138,10 @@ def deploy():
             f.write(build_commit)
 
         # Update runCI
-        runCI_repo = os.path.join(app.config['INSTALL_FOLDER'], 'install', 'ci-vm', 'ci-linux', 'ci', 'runCI')
-        runCI_nfs = os.path.join(config.get('SAMPLE_REPOSITORY', ''), 'vm_data', config.get(
```

Comment:
Why is it change from `app.config` --> `config`?

Reply:
app.config is not defined in current context I guess.